src/components/dashboard/PreviewItem.vue

```diff
@@ -21,8 +21,8 @@
       </div>
       <div v-else class="regular" :style="'background: '+colors.default_button">
         <div class="imageContainer" :style="'border-color: '+colors.default_button">
-          <b-img :src="getIconUrl(item.configuration.image_url)"/>
-        </div>`
+          <b-img :src="getIconUrl(item.configuration.image_url)" alt=""></b-img>
```

Comment:
A better way could be to change the `alt` attribute to `:alt="item.configuration.label"` or `:alt="item.configuration.label + ' logo'"`